.pre-commit-config.yaml

```diff
@@ -108,7 +108,7 @@ repos:
     hooks:
       - id: codespell
         additional_dependencies: ["tomli"]
-        args: [-L, lama]
+        args: ["-L", "lama", "--ignore-regex", ".{1024}|.*codespell-ignore.*"]
```

Comment:
This works so no change needed, but pyproject.toml also has an codespell ignore-words-list that can be used for false positives.

Reply:
Correct, but that ignore-words list is not related to just a single line or file, its for the entire project.